### PR TITLE
Improve "count" graphs for F5OS

### LINKED ISSUE #15334: F5OS Disk counts does not display correctly
### The problem

In 23.9.0 F5OS was added to the devices. I've discoved the devices correctly now, but on all three F5 OS rSeries devices the DISK count are counting upwards, instead of giving the current value. 

screenshot DISK count

Added discovery and poller output as text because of the size. 

[discovery output.txt](https://github.com/librenms/librenms/files/12657636/discovery.output.txt)
[poller output.txt](https://github.com/librenms/librenms/files/12657637/poller.output.txt)

### Output of ./validate.php

```txt
===========================================
Component | Version
--------- | -------
LibreNMS  | 23.9.1 (2023-09-19T03:59:06+02:00)
DB Schema | 2023_09_01_084057_application_new_defaults (259)
PHP       | 8.2.9
Python    | 3.9.16
Database  | MariaDB 10.5.16-MariaDB
RRDTool   | 1.7.2
SNMP      | 5.9.1
===========================================

[OK]    Composer Version: 2.5.4
[OK]    Dependencies up-to-date.
[OK]    Database connection successful
[OK]    Database Schema is current
[OK]    SQL Server meets minimum requirements
[OK]    lower_case_table_names is enabled
[OK]    MySQL engine is optimal
[OK]    Database and column collations are correct
[OK]    Database schema correct
[OK]    MySQl and PHP time match
[OK]    Distributed Polling setting is enabled globally
[OK]    Connected to rrdcached
[OK]    Active pollers found
[OK]    Dispatcher Service is enabled
[OK]    Locks are functional
[OK]    No python wrapper pollers found
[OK]    Redis is functional
[OK]    rrdtool version ok
[OK]    Connected to rrdcached
```

### What was the last working version of LibreNMS?

23.9.1

### Device info

F5OS F5 rSeries 2800

### Discovery output

```txt
Added the discovery output to discovery output.txt
```

### Poller output

```txt
Added the discovery output to poller output.txt
```

### PR DESCRIPTION
**Pull request by PipoCanaja**
Following #15334, change to Counter instead of Gauge for graphs. This will reset the graphs but the PR is fairly new and the current graphs are useless anyway so it won't be an issue I would say.

PR fixes #15241 and fixes #15334

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 15335`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by zagosch** [comment]
This fixes my problem, see: https://github.com/librenms/librenms/issues/15334#issuecomment-1727431300

File: includes/definitions/discovery/f5os.yaml [hunk 1]
```diff
@@ -124,60 +124,70 @@ modules:
...
+                    index: diskTotalIops0
                 -
                     oid: F5-PLATFORM-STATS-MIB::diskUtilizationStatsTable
                     value: F5-PLATFORM-STATS-MIB::diskReadIops
                     num_oid: .1.3.6.1.4.1.12276.1.2.1.2.2.1.5.8.112.108.97.116.102.111.114.109.7.110.118.109.101.48.110.49
                     descr: Read IOPS
                     group: Disk
-                    index: 1
+                    rrd_type: COUNTER
+                    index: diskReadIops1
```
Comment (murrant) [inline includes/definitions/discovery/f5os.yaml:136]: Why the number after index?

Comment (PipoCanaja) [reply includes/definitions/discovery/f5os.yaml:136]: There is no real reason for that indeed. Only copy-paste with right click in VI, no 'd' :D I can fix that.

Comment (PipoCanaja) [reply includes/definitions/discovery/f5os.yaml]: done

**Review by murrant — APPROVED** [review]

**Pull request merged**